web/src/ui/nodes/icons-and-colours.ts

```diff
@@ -151,3 +151,22 @@ export const executionMessageUI = (
 }
 
 // ---------------------------------------------------------
+
+// Provenance Highlight Colours ----------------------------
+
+export const provenanceHighlights = {
+  0: 'transparent',
+  1: '#f1f5fe', // blue-50
+  2: '#cfdefc', // blue-100
+  3: '#b1c9fa', // blue-200
+  4: '#92b3f7', // blue-300
+  5: '#77a0f5', // blue-400
+}
+
+export type ProvenanceHighlightLvl = keyof typeof provenanceHighlights
```

Comment:
You and your cutbacks! To save an old fart like me a few brain cycles could your neo-liberal fingers spare a couple more characters?
```suggestion
export type ProvenanceHighlightLevel = keyof typeof provenanceHighlights
```

Reply:
I'm surporised you didn't comment on this one: 
```
export const getProvHighlight = (miLvl: ProvenanceHighlightLvl) =>
```

Reply:
Neuseating